charts/rancher-csp-adapter/0.1.0+up0.1.0-rc5/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Chart.Name }}-status
+  namespace: cattle-csp-adapter-system

```

Comment:
Should be `{{ .Release.Namespace }}`

Reply:
I brought up same thing, but this is hardcoded in Rancher now: https://github.com/rancher/rancher/pull/37828/files

Reply:
Rancher needs the output produced by this deployment (which is a configmap). This needs to be in a specific namespace so that rancher can actually retrieve it. There are multiple ways of signaling this to rancher, but in order to ensure that the namespace is labeled as a part of the system project, rancher needs to know the namespace beforehand.